packages/driver/cypress/integration/e2e/uncaught_errors_spec.js

```diff
@@ -158,6 +158,19 @@ describe('uncaught errors', () => {
     cy.visit('/fixtures/global-error.html')
   })
 
+  it('global unhandledrejection', (done) => {
+    cy.once('uncaught:exception', (err) => {
+      expect(err.stack).contain('foo is not defined')
+      expect(err.stack).contain('one')
+      expect(err.stack).contain('two')
+      expect(err.stack).contain('three')
+
+      done()
+    })
+
+    cy.visit('/fixtures/unhandled-rejection.html')
+  })
+
```

Comment:
Should also add a test that `unhandled:rejection` actually fails the test if not handled, probably usin `cy.once('fail', (err) => { ... })`